src/main/java/me/totalfreedom/totalfreedommod/command/Command_undisguiseall.java

```diff
@@ -0,0 +1,30 @@
+package me.totalfreedom.totalfreedommod.command;
+
+import me.libraryaddict.disguise.DisallowedDisguises;
+import me.totalfreedom.totalfreedommod.rank.Rank;
+import me.totalfreedom.totalfreedommod.util.FUtil;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = Rank.SUPER_ADMIN, source = SourceType.BOTH)
+@CommandParameters(description = "Undisguise all players on the server", usage = "/<command>", aliases = "uall")
+public class Command_undisguiseall extends FreedomCommand
+{
+
+    @Override
+    public boolean run(CommandSender sender, Player playerSender, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
+    {
+        if (DisallowedDisguises.disabled)
```

Comment:
Won't this throw an error when LibsDisguises is not enabled on the server? You could instead check if `ldb.getLibsDisguisePlugin()` is null or not.